static/src/stylesheets/layout/_ab-new-header-test-variant.scss

```diff
@@ -136,19 +182,23 @@
     }
 }
 
-.new-header__burger-icon--open {
-    span {
-        &:nth-child(1) {
-            opacity: 0;
-        }
-        &:nth-child(2) {
-            transform: rotate(45deg);
-        }
-        &:nth-child(3) {
-            transform: rotate(-45deg);
-        }
-        &:nth-child(4) {
-            opacity: 0;
+.new-header__nav__menu-button--open {
+    transform: scale(1.1);
+
+    .new-header__burger-icon {
+        span {
```

Comment:
Great, makes sense. Think I am going to do a sperate PR for a different burger animation using befoe and after anyway.